lib/services/notification-api-service.js

```diff
@@ -43,18 +41,46 @@ function NotificationApiServiceFactory(
 
         if (_.has(message, 'nodeId')) {
             return self.postNodeNotification(message);
-        } else if (_.has(message, 'taskId')) {
-            if (_.has(message, 'progress')) {
-                // This will be progress update notification if taskId is specified
-                return self.postProgressNotification(message.taskId, message.progress);
-            }
```

Comment:
If you remove these lines, you need update Secure Erase job which still depends on this API.